### PR TITLE
Fix bug #524: support multiple interfaces on mac format

### PR DESCRIPTION
**Pull request by Rian83**
I got an issue (https://github.com/napalm-automation/napalm/issues/524)
```
* 3        f60c.be90.7a07    dynamic   300        F    F  Po2
  39       0100.5e00.4b4b    igmp      0          F    F  Po1 Po2 Po22
  516      0100.5e00.4b4b    igmp      0          F    F  Po1 Po2 Po14
  110      0100.5e00.0118    igmp      0          F    F  Po1 Po2 Po14
                                                          Po18 Po24
  110      0100.5e01.0203    igmp      0          F    F  Po1 Po2 Po24
  110      0100.5e01.0207    igmp      0          F    F  Po1 Po2 Po24
```
The 4th and 5th line will crash because : 
4th line : there's multiple interface at the end
5th line : the format of the line is not the same and is the suite from previous line

**What I do :** 
For support multiple interfaces for an unique MAC address: I assume to create a new dict key other_interfaces for keeping backward compatibility
FIX: mulitple interfaces on the same line
FIX: multiple interfaces on multiple lines

Previous output example : 
 `{u'vlan': 20, u'last_move': -1.0, u'active': False, u'mac': u'01:00:5E:7F:FF:FD', u'static': False, u'interface': u'Po1', u'moves': -1}`
New output example : 
`{u'vlan': 20, u'last_move': -1.0, u'active': False, u'mac': u'01:00:5E:7F:FF:FD', u'static': False, u'interface': u'Po1', u'moves': -1, u'other_interfaces': [u'Po2', u'Eth142/1/2']}`

**EDIT 1 :** 
no change of the model

### CODE REVIEW

**Comment by Rian83** [comment]
I've changed my previous output thanks to @itdependsnetworks 
Now I return the same output model but multiple entries for the same mac address if multiples interfaces exists.

**Comment by ktbyers** [comment]
@Rian83 Per comments on Slack I assume you are going to add some unit tests in for this?

File: napalm/nxos_ssh/nxos_ssh.py [hunk 1]
```diff
@@ -1243,13 +1243,20 @@ def get_mac_address_table(self):
...
+
         """
 
         #  The '*' is stripped out later
         RE_MACTABLE_FORMAT1 = r"^\s+{}\s+{}\s+\S+\s+\S+\s+\S+\s+\S+\s+\S+".format(VLAN_REGEX,
                                                                                   MAC_REGEX)
         RE_MACTABLE_FORMAT2 = r"^\s+{}\s+{}\s+\S+\s+\S+\s+\S+\s+\S+\s+\S+".format('-',
                                                                                   MAC_REGEX)
+        # REGEX dedicated for lines with only interfaces (suite of the previous MAC address)
+        RE_MACTABLE_FORMAT3 = r"^\s+"
```
Comment (ktbyers) [inline napalm/nxos_ssh/nxos_ssh.py]: I think you would need to expand this to at least:

RE_MACTABLE_FORMAT3 = r"\s+\S+"

**Comment by Rian83** [comment]
@ktbyers Yes I'll add a test for : 
- multiple interfaces on same line
- multiple interfaces on "new line"
- `+` at the begging of the line (because I don't see any test about it)

File: napalm/nxos_ssh/nxos_ssh.py (hunk 1 above)
Comment (Rian83) [reply napalm/nxos_ssh/nxos_ssh.py]: Yes you're right, I'll change it

**Comment by dbarrosop** [comment]
Thanks for this one, everything seems in order so I will proceed to merge :)

**Comment by Rian83** [comment]
You're welcome @dbarrosop, good news! First contribution

**Pull request merged**